### ISSUE TITLE
Logical Reading Order example

### ISSUE THREAD

**Issue by clapierre** [w3c/epub-specs#735]
EPUB A11Y Techniques
WCAG-006: Ensure Linear Reading Order of Publication

We should include a small sample here of what this looks like.

**Comment by mattgarrish** [comment]
There are a couple there now. There are still some other sections missing examples, as I was working on prose first.
